### PR TITLE
Add colemak patch

### PR DESCRIPTION
**Pull request by luukvbaal**

### CODE REVIEW

**Comment by jarun** [comment]
Thank you!

@anjerukare please confirm this works as intended.

**Comment by anjerukare** [comment]
Checked, works as intended!

But there's one thing. I didn't emphasize that these patches are for Colemak modification, which is called Colemak-DH.
Here are changes from Colemak: https://colemakmods.github.io/mod-dh/#colemak-changes.

To make patches for default Colemak layout, as I see it, we need to swap <kbd>m</kbd> with <kbd>h</kbd>.

**Comment by jarun** [comment]
@anjerukare can you please raise the PR to fix it? We are really busy.

**Comment by luukvbaal** [comment]
I think the question is do we want 2 new patches, 1 for "colemak" and 1 for "colemak-DH"...

**Comment by jarun** [comment]
Only one - "colemak".

**Comment by anjerukare** [comment]
> @anjerukare can you please raise the PR to fix it? We are really busy.

Same here. All I can do to help right now: [colemak-patches.tar.gz](https://github.com/jarun/nnn/files/9079126/colemak-patches.tar.gz).

**Comment by jarun** [comment]
@luukvbaal I think @anjerukare has forgotten that (s)he came up with the request.

I leave it up to you to update the patch or discard. If you want I can revert the earlier commit.

**Comment by luukvbaal** [comment]
I don't care, this seems a relatively free patch as in I don't think many conflicts will occur. I imagine it will be nice to have for colemak users but for that we need a colemak user to define the layout. I would say wait for a PR to update but if you want to make a release then idk.

**Comment by jarun** [comment]
I think it's fine not to update. I am sure users would have different preferences.

File: patches/check-patches.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ for ((n=1; n < z; ++n)); do
     done | tee "/dev/stderr" | (
         make clean -s
         xargs make 2>&1
-        if [ "$?" -ne 0 ]; then
+        if ! xargs make 2>&1; then
```
Comment (ggustafsson) [inline patches/check-patches.sh:21]: Line 20 should be removed, xargs make is executed twice.

Comment (luukvbaal) [reply patches/check-patches.sh:21]: Whoops, thanks.

**Pull request merged**